.env.example

```
@@ -48,4 +48,9 @@ ACESTEP_INIT_LLM=auto
 
 # ==================== API Server Settings ====================
 # API key for authentication (optional)
-# ACESTEP_API_KEY=sk-your-secret-key
\ No newline at end of file
+# ACESTEP_API_KEY=sk-your-secret-key
+
+# ==================== Startup Settings ====================
+# Skip model loading at startup (models will be lazy-loaded on first request)
+# Set to true to start server quickly without loading models
+# ACESTEP_NO_INIT=false
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)